File: apps/s-2-admin/src/api/team/TeamWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { BooleanFilter } from "../../util/BooleanFilter";
import { EventTypeListRelationFilter } from "../eventType/EventTypeListRelationFilter";
import { MembershipListRelationFilter } from "../membership/MembershipListRelationFilter";

export type TeamWhereInput = {
  id?: IntFilter;
  name?: StringNullableFilter;
  slug?: StringNullableFilter;
  logo?: StringNullableFilter;
  bio?: StringNullableFilter;
  hideBranding?: BooleanFilter;
  eventTypes?: EventTypeListRelationFilter;
  members?: MembershipListRelationFilter;
};

File: apps/cactus-demo-dotnet/src/APIs/Team/Dtos/TeamWhereInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class TeamWhereInput
{
    public int? Id { get; set; }

    public string? Name { get; set; }

    public string? Slug { get; set; }

    public string? Logo { get; set; }

    public string? Bio { get; set; }

    public bool? HideBranding { get; set; }

    public List<int>? EventTypes { get; set; }

    public List<int>? Members { get; set; }
}
